augur/validate.py

```diff
@@ -31,7 +30,7 @@ def load_json_schema(path):
     (located in augur/data)
     '''
     try:
-        schema = json.loads(resource_string(__package__, os.path.join("data", path)))
+        schema = json.loads(files(path).read_text(encoding="utf-8"))
```

Comment:
_non-blocking, just a note_

It doesn't matter for memory usage/read speed here, but we could use `augur.data.as_files()` as is here too:

```suggestion
        with as_file(path) as file, open(file, "r", encoding = "utf-8") as fh:
            schema = json.load(fh)
```

alternatively, keeping `augur.data.files()` but avoiding reading the whole file into memory:

```suggestion
        with files(path).open("r", encoding = "utf-8") as fh:
            schema = json.load(fh)
```

Reply:
Thanks, I went with the first suggestion since `files` is a bit confusing to me.